pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -324,6 +323,15 @@ func (s *srv) handleTraffic(c echo.Context) (err error) {
 					logger.WithError(err).Warn("Failed to send router configuration")
 					return err
 				}
+				stat := &ttnpb.GatewayStatus{
+					Time: receivedAt,
+					Versions: map[string]string{
+						"platform": fmt.Sprintf("%s:%s", version.Model, version.Station),
```

Comment:
This wasn't really the intention of the `versions` map (see API definition).

I couldn't find any documentation on the exact spec of these fields in the LNS documentation, but the CUPS documentation says:

> The `station`, `model`, and `package` fields describe the current version of Station, the gateway model, and the version of the package. The package version may contain information about the state of the underlying system and its configuration.

So `station` and `package` are version for sure, but `model` isn't. I suppose it would therefore make sense to take the `station` and `package` fields (and I guess also `firmware`) from the `version` message, and put those in the `Versions` map. We could put the `model` and `features` into the `Advanced` struct.

Reply:
```
  // - for example:
  //     firmware: "2.0.4"
  //     forwarder: "v2-3.3.1"
  //     fpga: "48"
  //     dsp: "27"
  //     hal: "v2-3.5.0"
```

The API doc only provides examples and isn't explicit about the keys so I chose my own. 
I've made the changes.